whylog/assistant/regex_assistant/guessing.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from whylog.assistant.pattern_object import ParamGroup, PatternObject
+from whylog.assistant.regex_assistant.regex import (
+    create_obvious_regex, regex_from_group_spans, verify_regex
+)
+from whylog.assistant.spans_finding import find_date_spans
+
+
+def guess_regex_objects(text):
```

Comment:
I will reneme variable to be more clear. regex object is more than regex bacause it keeps: line_text, regex, groups contents with converters. For now, this function returns regex objects that are instances of PatternObjects.